invideoquiz/public/js/src/invideoquiz.js

```diff
@@ -0,0 +1,109 @@
+/* Javascript for InVideoQuizXBlock. */
+function InVideoQuizXBlock(runtime, element) {
+    var videoId = $('.in-video-quiz-block').data('videoid');
+    var studentMode = $('.in-video-quiz-block').data('mode') !== 'staff';
+    var problemTimesMap = InVideoQuizXBlock.config[videoId] || {};
+    var extraVideoButton = '<button class="in-video-continue">Continue</button>';
+    var video;
+    var videoState;
+    var intervalTime = 500; // interval at which to check for problems to display
```

Comment:
We should explain why these values were chosen over others, i.e.: why not `1.0s`.